src/App.js

```diff
@@ -1,27 +1,18 @@
-import logo from './logo.svg';
+import logo from './logo.jpg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Based on the requirements of this project, the App.js component should be a class-based component. Let's kindly fix this.